docs/user/rstudio/ide/guide/code/code-navigation.qmd

```diff
@@ -40,15 +38,15 @@ The keyboard shortcut for Go to File/Function is <kbd>Ctrl+.</kbd> which will op
 
 ## Go to Function Definition
 
-Since an index of your R source code (as described above) is maintained, RStudio can also help you quickly navigate to the definition of any R function. To navigate a function definition you place your cursor on the function name (it doesn't have to be fully selected) and then from the RStudio menu choose the **Code** \> **Go to Function Definition** command.
+Since an index of the current Project's source code (as described above) is maintained, RStudio can also help with quick navigation to the definition of any R function. To navigate a function definition, place the cursor on the function name (it doesn't have to be fully selected), and then from the RStudio menu choose the **Code** \> **Go to Function Definition** command.
 
-From either the source editor or the console, you can also access go to function definition :
+From either the source editor or the console, the following shortcuts provide access to the function definition:
 
 -   Using the **F2** keyboard shortcut
 
 -   Using **Ctrl+Click** with the mouse
 
```

Comment:
is it cmd + click on a mac? F2 shortcut works for me but can't get the other one to work

Reply:
Download the daily build at cmd + click works for me! cntrl + click doesn't so will want to include the mac shortcut.

Reply:
Using <kbd>Ctrl+Click</kbd> (<kbd>Cmd+Click</kbd> on Mac) with the mouse